src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -144,6 +155,15 @@ export default {
       var jointlockMsg = new ROSLIB.Message(jointData);
       this.jointlock_pub.publish(jointlockMsg);
     },
+
+    updateSlowMode: function (enabled) {
+      this.slow_mode = enabled;
+      const slowData = {
+        data: this.slow_mode
+      };
+      var slowmodeMsg = new ROSLIB.Message(slowData);
+      this.slowmode_pub.publish(slowmodeMsg);
```

Comment:
small nit but slowmode_pub and slowmodeMsg should be renamed to slow_mode_pub and slowModeMsg